### PR TITLE
Allow creating an account password policy within a new AWS account.

### PR DESCRIPTION
**Pull request by sgielen**

### CODE REVIEW

File: modules/avm/variables.tf [hunk 1]
```diff
@@ -140,3 +140,49 @@ variable "trigger_prefixes" {
   default     = ["modules"]
   description = "List of repository-root-relative paths which should be tracked for changes"
 }
+
```
Comment (fgoncalves-io) [inline modules/avm/variables.tf:143]: I would suggest changing all these variables into a single object...similar to [this one](https://github.com/schubergphilis/terraform-aws-mcaf-landing-zone/blob/master/variables.tf#L73), for example.

**Review by fgoncalves-io — DISMISSED** [review]

File: modules/avm/variables.tf (hunk 1 above)
Comment (sgielen) [reply modules/avm/variables.tf:143]: Done!

File: .github/workflows/terraform.yml [hunk 2]
```diff
@@ -47,6 +47,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: ${{ github.event.pull_request.head.ref }}
+        continue-on-error: true # added this to prevent a PR from a remote fork failing the workflow
```
Comment (stefanwb) [inline .github/workflows/terraform.yml:50]: As discussed offline, tnx for adding this

**Review by stefanwb — APPROVED** [review]
Can you add the `continue-on-error: true # added this to prevent a PR from a remote fork failing the workflow` line to the check out step of terraform-docs in

**Comment by stefanwb** [comment]
@sgielen has updated actions on my request

**Pull request merged**